aspnetcore/client-side/angular2.md

```diff
@@ -137,52 +140,52 @@ As was previously mentioned, the Webpack dev middleware eliminates the need to r
 
-These commands will produce development-grade builds. If you want to produce production-grade builds, then also pass the flag `--env.prod` when invoking Webpack.
+~These commands will produce development-grade builds. If you want to produce production-grade builds, then also~ For production, pass the flag `--env.prod` when invoking Webpack.
 
-### Hot Module Replacement (HMR)
+## Hot Module Replacement (HMR)
 
-This feature can be enabled while configuring the Webpack dev middleware. It watches the changes made to the code or markup and pushes them directly into the running application without requiring a reload of the whole page. It is especially useful while in the middle of a debugging session, since a full page reload deletes the application state and would force you to start over.
+This feature can be enabled while configuring the Webpack dev middleware. It watches the changes made to the code or markup and pushes them ~directly~ into the running application without requiring a reload of the whole page. It is especially useful while in the middle of a debugging session, since a full page reload deletes the application state and would force you to start over.
 
-To enable HMR, modify the Webpack dev middleware component's configuration in your `Startup.cs` file's `Configure` method. It should look as follows:
+To enable HMR, modify the Webpack dev middleware component's configuration in the `Configure` method. It should look as follows:
 
 [!code-csharp[Main](../client-side/angular2/sample/DemoUsingAngular/Startup.cs?range=44-46)]
 
 ## Prerequisites for using SpaServices
-To work with SpaServices, first make sure you’ve installed the following:
-* .NET Core SDK 1.0 RC4 (or later) for Windows, Mac, or Linux
-* If you’re on Windows, you can install the latest Visual Studio 2017, which includes it. Be sure you have VS 2017 build 26206 or later — older versions won’t work.
-* Node.js, version 6 or later
+~To work with SpaServices, first make sure you’ve installed the following:~
+* .NET Core SDK 1.0 ~RC4~ or  (or later) for Windows, Mac, or Linux [What - and make this a link]
+* If you’re on Windows, you can install the latest Visual Studio 2017 {make this a link}, which includes SpaServices. ~Be sure you have VS 2017 build 26206 or later — older versions won’t work.~
+* Node.js, version 6 or later {link?}
 
 ## Creating an Angular application using SpaServices
 
-As a starting point, JavaScriptServices provides pre-configured application templates. SpaServices is used in these templates, in conjunction with different frameworks and libraries including Angular, Aurelia, Knockout, React, and Vue.
+~As a starting point,~ JavaScriptServices {link here} provides pre-configured application templates. SpaServices is used in these templates, in conjunction with different frameworks and libraries such as Angular, Aurelia, Knockout, React, and Vue.
```

Comment:
@Rick-Anderson Is it overkill to provide a link for JavaScriptServices here, since the reference to it in the **What is SpaServices** section already provides the link?